### ISSUE TITLE
Add switches and AP's as sensor

### ISSUE THREAD

**Issue by renedis** [custom-components/sensor.unifigateway#9]
Request here.

The UniFi API provides al metrics/sensors (even DPI info!). I've searched in the API using the https://github.com/Art-of-WiFi/UniFi-API-browser and found out that also switches and AP's show there CPU usage, CPU load 1|5|15, Memory usage, Uptime, Temperature etcetera.

It would be nice if these are added to this plugin.

Examples:

**Comment by jchasey** [comment]
Agreed this info is there, but this component is focussed on the USG as opposed to those other devices. The question would become where to stop in terms of adding all the extra info...

**Comment by ciechompl** [comment]
I would kile to have the same as @renedis ;-)  quick vie on the CPU&MEM each unifi device.  
Very useful information and even if you @jchasey say that this component is focused on USG , please please please add this info for us :) 
Can be done as separate "system-stats" sensor listing in attributes all unifi devices of even under usg or in any other place you like.

**Comment by arjones5** [comment]
I'm jumping on the bandwagon - this is an awesome app for the USG. As a dev IRL, @jchasey has a point...where do you draw the line between a USG app and a UniFi Controller pseudo interface? Playing devil's advocate, been messing with the code to get some of the other metrics I need and we can probably spin this into a UniFi sensor. My switched run hot (like I'm sure everyone elses do) and it's be nice to just dump the info into lovelace instead of taking the 5 seconds to log into the controller.

**Comment by JamesMarcum** [comment]
https://github.com/w1tw0lf/Unifi-Device-info might solve this problem?
